internal/wgnl/configure_linux.go

```diff
@@ -210,26 +211,27 @@ func sockaddrBytes(endpoint net.UDPAddr) ([]byte, error) {
 		var addr [16]byte
 		copy(addr[:], endpoint.IP.To16())
 
-		sa := unix.RawSockaddrInet6{
-			Family: unix.AF_INET6,
-			Port:   uint16(endpoint.Port),
-			Addr:   addr,
-		}
+		buf := bytes.NewBuffer(nil)
+		binary.Write(buf, binary.LittleEndian, uint16(unix.AF_INET6))
```

Comment:
This won't be correct on big endian architectures, right?  You can do `ne := nlenc.NativeEndian()` and pass that for the fields that are endian-specific.